tests/sentry/api/serializers/test_fields.py

```diff
@@ -30,14 +31,18 @@ def test_allow_null(self):
 
         serializer = DummySerializer(data=data)
         assert not serializer.is_valid()
-        assert serializer.errors == {"a_field": ["This field may not be null."]}
+        assert serializer.errors == {
+            "a_field": [ErrorDetail(string="This field may not be null.", code="null")]
+        }
```

Comment:
These changes are just internal I assume, and we still produce the output via the api?

Reply:
Yeah.